### PR TITLE
[NUI] Fix Dispose warning error[CA1001]

### PR DESCRIPTION
**Pull request by huiyueun**
CA100: Types that own disposable fields should be disposable
https://docs.microsoft.com/ko-kr/dotnet/fundamentals/code-analysis/quality-rules/ca1001

Signed-off-by: huiyu.eun <huiyu.eun@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/Accessibility/Accessibility.cs [hunk 1]
```diff
@@ -30,20 +30,16 @@ namespace Tizen.NUI.Accessibility
...
         {
             dummy = new View();
             dummy.Name = "dali-atspi-singleton";
         }
-        /// <summary>
-        /// destructor. This is HiddenAPI. recommended not to use in public.
-        /// </summary>
-        ~Accessibility()
-        {
-        }
```
Comment (WonyoungChoi) [inline src/Tizen.NUI/src/public/Accessibility/Accessibility.cs:46]: Need destructor in dispose pattern
```csharp
~Accessibility() {
  Dispose(false);
}
```

Comment (huiyueun) [reply src/Tizen.NUI/src/public/Accessibility/Accessibility.cs:46]: I've fixed patch.

**Comment by WonyoungChoi** [comment]
introducing new base class does not make ABI break. you can ignore the `ACR Required` label in this case.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**